packages/manager/apps/cloud/client/index.html

```diff
@@ -6,6 +6,8 @@
         <title>OVH - Cloud</title>
```

Comment:
```suggestion
        <title>OVHcloud - Cloud</title>
```
🤔

Reply:
Thanks @FredericEspiau! You're right!

I think we can take a look at [`@ovh-ux/ng-ui-router-title`](https://github.com/ovh/manager/tree/develop/packages/components/ng-ui-router-title) to handle the title regarding the navigation.